src/explore-education-statistics-admin/src/pages/bau/BauImportsPage.tsx

```diff
@@ -0,0 +1,94 @@
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import React from 'react';
+import importStatusBauService from '@admin/services/importStatusBauService';
+
+const BauImportsPage = () => {
+  const { value, isLoading } = useAsyncRetry(() =>
+    importStatusBauService.getAllIncompleteImports(),
+  );
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Incomplete imports' },
+      ]}
+    >
+      <h1 className="govuk-heading-xl">
```

Comment:
Set page title and caption using's `Page` `caption` and `title` props. These have the added benefit of updating the browser page title as well.